viewing_party/party.py

```diff
@@ -1,23 +1,174 @@
+"""
+Dear Ashley (or Instructor),
+
+Can you please take a look at the comments in my wave 3.
+I tried doing something that didn't work out and so had to 
+abandon it all together. I can't figure out what the problem
+is and would appreciate a look.
+
+Also, answering my questions on Wednesday!
+And WOOT WOOT for finishing something I thought would even 
+be able to do halfway!!!!!
+
+Thank you so much,
+Huma Hameed
+(she/her)
+"""
+
+
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    for key, value in new_movie.items():
+        print(new_movie)
+        if value == None:
+            return None
+    return new_movie

```

Comment:
Nice work!

We can use a guard clause at the very beginning of your function. Right now, your function makes a dictionary then check if the values in the dictionary are None. 

If we use a guard clause, then we can just check if title/genre/rating are Falsy (empty lists) and return None without even creating a dictionary and checking it. That would look like: 
```
if not title and not genre and not rating: 
      return None
```

After this guard clause, then you can create your dictionary right after. Then you could remove lines 28 - 31.